web/packages/teleport/src/Integrations/Enroll/AwsConsole/Access/Access.tsx

```diff
@@ -0,0 +1,255 @@
+        </Alert>
+        <ButtonPrimary
+          onClick={() =>
+            history.push(
+              cfg.getIntegrationEnrollRoute(
+                IntegrationKind.AWSRa,
+                'integration'
+              )
+            )
+          }
+          width="100px"
+        >
+          Back
+        </ButtonPrimary>
+      </FeatureBox>
+    );
+  }
+
+  return (
+    <>
+      <Flex mt={3} justifyContent="space-between" alignItems="center">
+        <H2>Configure Access</H2>
+        <InfoGuideButton
+          config={{
+            guide: <Guide resourcesRoute={resourceRoute} />,
+          }}
+        />
+      </Flex>
+      <P2 mb={3}>
+        Import and synchronize AWS IAM Roles Anywhere Profiles into Teleport.
+        Imported Profiles will be available as Resources with each Role
+        available as an account.
+      </P2>
+      {status === 'error' && (
+        <Alerts.Danger details={error.message}>
+          Error: {error.name}
+        </Alerts.Danger>
+      )}
+      {status === 'pending' && (
+        <Box data-testid="loading" textAlign="center" m={10}>
+          <Indicator />
+        </Box>
+      )}
+      {status === 'success' &&
+        (!data.profiles || data.profiles.length === 0) && (
+          <ProfilesEmptyState />
+        )}
+      {status === 'success' && data.profiles.length > 0 && (
+        <Profiles
+          data={data.profiles}
+          fetchStatus={isFetching ? 'loading' : ''}
+          filters={filters}
+          setFilters={setFilters}
+          refetch={refetch}
+          syncAll={syncAll}
+          setSyncAll={setSyncAll}
+        />
+      )}
+      {submitSync.error && (
+        <Alerts.Danger details={submitSync.error?.message} mt={2}>
+          Error: {submitSync.error.name}
+        </Alerts.Danger>
+      )}
+      <Flex gap={3} mt={3}>
+        <ButtonPrimary width="200px" onClick={() => submitSync.mutate()}>
+          Enable Sync
+        </ButtonPrimary>
+        <ButtonSecondary
+          onClick={() =>
+            history.push(
+              cfg.getIntegrationEnrollRoute(
+                IntegrationKind.AWSRa,
+                'integration'
+              )
+            )
+          }
+          width="100px"
+        >
+          Back
+        </ButtonSecondary>
+      </Flex>
+    </>
+  );
+}
+
+function ProfilesEmptyState() {
+  return (
+    <CardTile alignItems="center" gap={4}>
+      <ResourceIcon name="awsidentityandaccessmanagementiam" width="164px" />
+      <Flex flexDirection="column" alignItems="center">
+        <H3 mb={1}>No AWS IAM Roles Anywhere Profiles Found</H3>
+        <P2>Create AWS IAM Roles Anywhere Profiles in your AWS console</P2>
+      </Flex>
+      <Flex gap={3}>
+        <ButtonPrimary as="a" target="blank" href={rolesAnywhereCreateProfile}>
+          Create AWS Roles Anywhere Profiles
+        </ButtonPrimary>
+        <ButtonBorder intent="primary">
+          Refresh AWS Roles Anywhere Profiles
```

Comment:
lol, i'm sure these weren't your wording... but the times i read `AWS Roles Anywhere` 🤣 

might we decrease it a bit? Can we just say `No Profiles Found` and get rid of `<P2>Create....</P2>` since the primary button already says it?

and for refresh button, maybe just `Refresh Profiles`?